benchmarks/timeseries/benchmark_datasourcing.py

```diff
@@ -31,10 +31,10 @@
     from tests.utils import MockMicrogridClient
 except ImportError:
     print(
-        "Error: Unable to import the `tests` package. "
+        "\nError: Unable to import the `tests` package. "
```

Comment:
Why the extra leading `\n` here? Looks weird.

Reply:
It just looked a lot nicer in my validate script output :innocent: but eventually figured we don't need to check for docu-wrapped examples in `benchmarks` or `examples` so I can just remove it again, too